csv_detective/detect_labels/geo/lonlat_wgs/__init__.py

```diff
@@ -0,0 +1,43 @@
+from csv_detective.parsing.text import header_score
+
+PROPORTION = 0.5
+
+
+def _is(header: str) -> float:
```

Comment:
Do we risk a conflict on all overrides between words_combinations_list in lonlat and latlon?
Should we have a common list?